extensions/python/src/main/resources/jet_to_python_init.sh

```diff
@@ -5,7 +5,7 @@ set -e -x
 
 python3 -m venv jet_to_python_env --system-site-packages
 source jet_to_python_env/bin/activate
-python3 -m pip install protobuf grpcio
+python3 -m pip install protobuf==3.11.3 grpcio==1.26.0
```

Comment:
I think it should be possible to use maven properties here and add a resource transformation step

Reply:
see for example `jet-runtime.properties`

Reply:
I see for example in python module we use proto 3.9 and 3.11 here and also different grpc versions

Reply:
That's updated as well.